src/Lean/Server/FileWorker/SemanticHighlighting.lean

```diff
@@ -120,10 +256,12 @@ partial def collectSyntaxBasedSemanticTokens : (stx : Syntax) → Array LeanSema
       return tokens
     return tokens.push ⟨stx, keywordSemanticTokenMap.getD val .keyword⟩
 
+
+open Lean.Doc.Syntax in
 /-- Collects all semantic tokens from the given `Elab.InfoTree`. -/
 def collectInfoBasedSemanticTokens (i : Elab.InfoTree) : Array LeanSemanticToken :=
-  List.toArray <| i.deepestNodes fun _ i _ => do
-    let .ofTermInfo ti := i
+  Array.flatten <| List.toArray <| i.deepestNodes fun _ info _ => do
```

Comment:
I assume this entire refactor of `collectInfoBasedSemanticTokens` is a remnant of something that was removed later on?

Reply:
Indeed. Sorry. Will revert.